### PR TITLE
Added accessor functions for SubMesh._materialDefines

### PR DESCRIPTION
**Pull request by Popov72**
See: https://forum.babylonjs.com/t/how-to-override-materials/6721/29

### CODE REVIEW

**Comment by Popov72** [comment]
I don't understand the documentation error as I did update the what's new.md file...

File: src/Meshes/subMesh.ts [hunk 1]
```diff
@@ -28,6 +28,21 @@ export class BaseSubMesh {
...
+    /**
+     * Gets material defines used by the effect associated to the sub mesh
+     */
+    public get materialDefines(): Nullable<MaterialDefines> {
+        return this._materialDefines;
+    }
+
+    /**
+     * Sets material defines used by the effect associated to the sub mesh
+     * @param defines defines the set of defines used to compile the effect
```
Comment (deltakosh) [inline src/Meshes/subMesh.ts]: You need to remove the @param here

Comment (Popov72) [reply src/Meshes/subMesh.ts]: Thanks, it is corrected.

**Comment by deltakosh** [comment]
Thanks!

**Pull request merged**